CRM/Core/Payment.php

```diff
@@ -727,6 +765,8 @@ protected function getDirectDebitFormFields() {
    *
    * @return array
    *   field metadata
+   *
+   * @throws \Exception
```

Comment:
We should throw a CRM_Core_Exception - it might be something lower done that needs fixing - just don't add the throws if you don't want to deal with that & we'll get it another time

Reply:
Not sure -but you've added throws Exception to the comment block & if that is true it shouldn't be (it should be some type of exception) - but you can just not add to the comment block for this PR